### PR TITLE
docs - add filter customization

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Notes on filter customization with dropdown lists and search boxes for the native editor.

Did some cleanup while I was in there. There's a case for revamping this whole section (e.g., splitting field filters into their own page and maybe combining it with the learn article), but wanted to get this change in first.

### CODE REVIEW

File: docs/questions/native-editor/sql-parameters.md [hunk 1]
```diff
@@ -324,16 +221,123 @@ WHERE
 {% endraw %}
 ```
 
-### SQL syntax
+## Customizing dropdown lists and search box values
```
Comment (jeff-bruemmer) [inline docs/questions/native-editor/sql-parameters.md:224]: @abdhabli This is the only section you really need to review, as the other stuff is just moving around.

File: docs/questions/native-editor/sql-parameters.md [hunk 2]
```diff
@@ -324,16 +221,125 @@ WHERE
...
+
+With Text and Field filter variables, you can tell Metabase what values people can choose from when using a filter with a dropdown list or search box.
 
-Make sure your SQL dialect matches the database you've selected. Common errors:
+1. In the native editor, add a {% raw %}{{variable}}{% endraw %} in double braces.
+2. If the sidebar doesn't open, you can click on the **{x}** icon on the right to open the **Variables** sidebar.
+3. In the **Settings** tab, set the **Variable type** to either "Text" or "Field Filter".
+4. Scroll down to **How should users filter on this variable?** Pick either **Dropdown list** or **Search box**.
```
Comment (abdhabli) [inline docs/questions/native-editor/sql-parameters.md:231]: I think the " Scroll down" doesn't fit here, as the sidebar options are fully visible without the need to scroll

Screenshot 2024-03-11 at 12 09 52 PM

File: docs/questions/native-editor/sql-parameters.md [hunk 3]
```diff
@@ -324,16 +221,125 @@ WHERE
...
+6. Metabase will pop up a modal where you can select **Where the values should come from**.
 
-| Database | Do this                    | Avoid                |
-| -------- | -------------------------- | -------------------- |
-| BigQuery | `` FROM `dataset.table` `` | `FROM dataset.table` |
-| Oracle   | `FROM "schema"."table"`    | `FROM schema.table`  |
+You can choose:
 
-For more help, see [Troubleshooting SQL error messages](../../troubleshooting-guide/error-message.md#sql-editor).
+- A custom list. Enter each item on a line.
```
Comment (abdhabli) [inline docs/questions/native-editor/sql-parameters.md:237]: I think this should follow the second one "A model or saved question", as that is the sequence presented in the app 
Screenshot 2024-03-11 at 12 18 14 PM

**Review by abdhabli — APPROVED** [review]
added small comments, thank you Jeff 👍

**Review by abdhabli — APPROVED** [review]

**Pull request merged**